ui/raidboss/data/05-shb/eureka/bozjan_southern_front.js

```diff
@@ -1113,5 +1142,84 @@ export default {
         'Winds\' Peak': '超级烈风波',
       },
     },
+    {
+      'locale': 'ko',
+      'missingTranslations': true,
+      'replaceSync': {
+        '4Th Legion Helldiver': 'IV군단 헬다이버',
+        'Adrammelech': '아드람멜렉',
+        'Bladesmeet': '검들의 대광장',
+        'Brionac': '브류나크',
+        'Dawon': '다우언',
+        'Eaglesight': '독수리 광장',
+        'Lightsphere': '빛 구체',
+        'Lyon The Beast King': '마수왕 라이언',
+        'Majesty\'s Auspice': '단상',
+        'Shadowsphere': '그림자 구체',
+        'The airship landing': '골드 소서 비공정 승강장',
+        'The grand gates': '성문',
+        'Verdant Plume': '진녹색 날개',
+      },
+      'replaceText': {
+        '(?<!Command: )Chain Cannon': '기관총 일제 발사',
+        '(?<!Command: )Dive Formation': '대물 일제 돌격',
+        '(?<!Command: )Infrared Blast': '대물 열선',
+        '(?<!Command: )Lateral Dive': '대물 돌진 공격',
+        'Accursed Becoming': '마법 합성',
+        'Aero IV': '에어로쟈',
+        'Anti-Warmachina Weaponry': '마도 병기 대응 공격',
+        'Blizzard IV': '블리자쟈',
+        'Burst II': '버스트',
+        'Call Beast': '부르기',
+        'Command: Chain Cannon': '지령: 기관총 일제 발사',
+        'Command: Dive Formation': '지령: 대물 일제 돌격',
+        'Command: Infrared Blast': '지령: 대물 열선',
+        'Command: Joint Attack': '지령: 대물 집중 공격',
+        'Command: Lateral Dive': '지령: 대물 돌진 공격',
+        'Command: Suppressive Formation': '대인 제압 돌격',
+        'Curse Of The Fiend': '마법인',
+        'Electric Anvil': '전기 모루',
+        'Energy Generation': '에너지 구체 생성',
+        'Explosion': '폭산',
+        'False Thunder': '인공 번개',
+        'Fervid Pulse': '열렬한 고동',
+        'Fire IV': '파이쟈',
+        'Flare': '플레어',
+        'Frigid Pulse': '냉랭한 고동',
+        'Heart Of Nature': '지령맥',
+        'Holy IV': '홀리쟈',
+        'Lightburst': '빛 분출',
+        'Lightning Shower': '번개 세례',
+        'Magitek Magnetism': '마도 자석',
+        'Magitek Missiles': '마도 미사일',
+        'Magnetic Jolt': '자력 간섭',
+        'Meteor': '메테오',
+        'Molting Plumage': '털갈이',
+        'Mrv Missile': '다탄두 미사일',
+        'Nature\'s Blood': '파도지령참',
+        'Nature\'s Pulse': '파도지령충',
+        'Obey': '명령 따르기',
+        'Pentagust': '다섯 갈래 돌풍',
+        'Polar Magnetism': '자석 변환',
+        'Pole Shift': '자기장 전환',
+        'Raging Winds': '풍렬비상류',
+        'Ready': '명령하기',
+        'Scratch': '생채기',
+        'Shadow Burst': '그림자 폭발',
+        'Shock': '방전',
+        'Stone IV': '스톤쟈',
+        'Surface Missile': '대지 미사일',
+        'Swooping Frenzy': '광란의 급강하',
+        'Taste Of Blood': '귀곡혈산참',
+        'The King\'s Notice': '패왕사시안',
+        'Thunder IV': '선더쟈',
+        'Tornado': '토네이도',
+        'Twin Agonies': '쌍마사왕참',
+        'Voltstream': '번개 급류',
+        'Warped Light': '섬광포',
+        'Water IV': 'ウォタジャ',
```

Comment:
```suggestion
        'Water IV': '워터쟈',
```

Reply:
Thanks for spotting. The ID of Water IV is '53D8' which is all japanese except for the english translation.